src/Mod/TechDraw/Gui/QGTracker.cpp

```diff
@@ -437,13 +438,16 @@ void QGTracker::setPoint(std::vector<QPointF> pts)
         Base::Console().Message("QGTracker::setPoint - no pts!\n");
         return;
     }
-    prepareGeometryChange();
-    QPainterPath newPath;
-    QPointF center = pts.front();
-    double radius = 50.0;
-    newPath.addEllipse(center, radius, radius);
-    setPath(newPath);
-    setPrettyNormal();
+    constexpr float baseVertexSize = 7.0F;
+
+    auto point = new QGIVertex(-1);
+    point->setParentItem(this);
+    point->setPos(pts.front());
+    point->setRadius(static_cast<float>(baseVertexSize * Preferences::vertexScale()));
+    point->setNormalColor(Qt::blue);
+    point->setFillColor(Qt::blue);
```

Comment:
Why hardcoded Qt::blue?  There are preferences in PreferencesGui.cpp that could be used as in QGIViewPart::drawAllVertexes():

```
    auto vp(static_cast<ViewProviderViewPart*>(getViewProvider(getViewObject())));
    float lineWidth = vp->LineWidth.getValue() * lineScaleFactor;     //thick
    double vertexScaleFactor = Preferences::getPreferenceGroup("General")->GetFloat("VertexScale", 3.0);
    QColor vertexColor = PreferencesGui::getAccessibleQColor(PreferencesGui::vertexQColor());
```

Reply:
@wandererfan, I suggested using blue for not yet added geometry - as when adding a new leader line, it is blue. It is for distinguishing the vertex until finally added by task finish

Reply:
Yeah, if both the regular vertices and the picked point use the same colour then it might be confusing. Though, maybe we can add another preference for colour of temporary geometry or something?

Reply:
So, the current approach of the using the blue colour as a differentiator is good? Do we also need a preference for this then?

Reply:
It would be nice to have a preference.  There is a accessibility preference for light-on-dark pages - not sure how blue would work in this case.  May need the ability to change the colour.

Reply:
Ok then a preference for the colour it is.
Can  you also please answer https://github.com/FreeCAD/FreeCAD/pull/20095#discussion_r1988024158?

Reply:
@captain0xff I suggest making the preference in a seperate PR for not stalling this PR.